app.js

```diff
@@ -0,0 +1,5 @@
+var warning = document.getElementById("warning");
```

Comment:
Please avoid using `var` as it's value can be reassigned outside the block scope and this may get problematic when working on a large code base.

**Rule of thumb is**
- Always prioritize using `const` if you know the value won't get reassigned to avoid bugs as it also helps other devs you might be working with in a codebase for readability
- Always use `let` as much as possible than using `var` if you know the value will get reassigned and `let` only works on it's block scope

```suggestion
const warning = document.getElementById("warning");
```